python/paddle/fluid/layers/nn.py

```diff
@@ -1273,7 +1273,7 @@ def sequence_conv(input,
     return helper.append_activation(pre_act)
 
 
-def sequence_softmax(input, param_attr=None, bias_attr=None, use_cudnn=True):
+def sequence_softmax(input, param_attr=None, bias_attr=None, use_cudnn=False):
```

Comment:
Note: for sequence_softmax, the implementation of cudnn is slow, because the number of call cudnn is equal to the number of squence, it is time-consuming.